pkg/policy/mutate/validate.go

```diff
@@ -17,22 +18,28 @@ import (
 
 // Mutate provides implementation to validate 'mutate' rule
 type Mutate struct {
-	mutation    kyvernov1.Mutation
-	authChecker auth.AuthChecks
+	rule                  *kyvernov1.Rule
+	mutation              kyvernov1.Mutation
```

Comment:
You have the entire `rule` already, why another `mutation` field?

Reply:
@realshuting fixed